src/app/kbv/controllers/question.js

```diff
@@ -0,0 +1,125 @@
+const dynamicTranslate = require("../../../lib/dynamic-i18n");
+const axios = require("axios");
+const BaseController = require("hmpo-form-wizard").Controller;
+
+const debug = require("debug")("app:simplified:question:ctrl");
+
+const {
+  API: {
+    BASE_URL,
+    PATHS: { QUESTION, ANSWER },
+  },
+} = require("../../../lib/config");
+
+class QuestionController extends BaseController {
+  configure(req, res, next) {
```

Comment:
Does configure(..) need to follow the same signature as `saveValues()` and `locals()`

Reply:
https://github.com/HMPO/hmpo-form-wizard/blob/master/lib/controller/index.js#L117-L131

Configure gives you an empty method to override - so it's easy :)